components/camera/videosource/webcam.go

```diff
@@ -58,9 +52,6 @@ func init() {
 		resource.Registration[camera.Camera, *WebcamConfig]{
 			Constructor: NewWebcam,
 		})
-	if err := json.Unmarshal(intrinsics, &data); err != nil {
-		logging.Global().Errorw("cannot parse intrinsics json", "error", err)
-	}
```

Comment:
do we just not use this json anymore?

Reply:
Correct -- sorry for the delay. The only usage of `data` was removed in [this commit](https://github.com/viamrobotics/rdk/commit/827a907748201d1582948804c4613b2817a73e1f#diff-08d95bbf0beb16baaedca5573e2bd015bcf67e778542018aa23c6f6dda98ceecL636)